gammapy/data/tests/test_data_store.py

```diff
@@ -58,6 +58,16 @@ def test_DataStore_load(data_manager):
     events = data_store.load(obs_id=23037, filetype='events')
     assert isinstance(events, EventList)
 
+@pytest.mark.xfail
+@requires_data('hess')
+def test_DataStore_load_all(data_manager):
```

Comment:
When I run the test python setup.py test -V -t gammapy/data/tests/test_data_store.py, I got these failures:
```
collected 5 items 

gammapy/data/tests/test_data_store.py::test_DataStore_construction xfail
gammapy/data/tests/test_data_store.py::test_DataStore_filenames xfail
gammapy/data/tests/test_data_store.py::test_DataStore_load xfail
gammapy/data/tests/test_data_store.py::test_DataStore_load_all xfail
gammapy/data/tests/test_data_store.py::test_DataStore_other xfail

========================== 5 xfailed in 1.30 seconds ===========================

```
Since it appears also for other test I don"t know if the test failes ou if this is a kind of warning?